homeassistant/components/blebox/sensor.py

```diff
@@ -56,10 +63,55 @@
         native_unit_of_measurement=PERCENTAGE,
     ),
     SensorEntityDescription(
-        key="wind_speed",
+        key="wind",
         device_class=SensorDeviceClass.WIND_SPEED,
         native_unit_of_measurement=UnitOfSpeed.METERS_PER_SECOND,
     ),
+    SensorEntityDescription(
+        key="illuminance",
+        device_class=SensorDeviceClass.ILLUMINANCE,
+        native_unit_of_measurement=LIGHT_LUX,
+    ),
+    SensorEntityDescription(
+        key="forwardActiveEnergy",
+        device_class=SensorDeviceClass.ENERGY,
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+    ),
```

Comment:
As per our guidelines, please split out dependency bumps and new features in different PRs as much as possible.

../Frenck

Reply:
OK, but unfortunately I will have to keep changes to `key="powerConsumption"` and  `key="wind"` when updating to v2.4.2. 

Does that count as "as much as possible"?

Reply:
If those are necessary changes to keep everything functioning as is: Yes, that is fine.

Reply:
Done, removed almost everything except two lines mentioned earlier as this is due to the downstream change.